src/olympia/addons/serializers.py

```diff
@@ -152,21 +154,27 @@ class AddonSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Addon
-        fields = ('id', 'authors', 'average_daily_users', 'current_version',
-                  'default_locale', 'description', 'edit_url', 'guid',
-                  'homepage', 'icon_url', 'is_disabled', 'is_experimental',
-                  'is_listed',
-                  'is_source_public', 'name', 'last_updated', 'previews',
-                  'public_stats', 'ratings', 'review_url', 'slug', 'status',
-                  'summary', 'support_email', 'support_url', 'tags',
-                  'theme_data', 'type', 'url', 'weekly_downloads')
+        fields = ('id', 'authors', 'average_daily_users', 'categories',
```

Comment:
This seems a bit annoying to change and harder to scan than a one-item-per-line list. Would converting to a list like

```python
'id',
'authors',
[...]
```

be okay?